### PR TITLE
Update Engineering Diffraction instructions

### LINKED ISSUE #35052: Update testing instructions for Engineering Diffraction
- [x] There are no GSASII tab instructions.
- [x] Test 2 fix indent of point 2. 
- [x] Test 3, points 8+9 say to repeat steps after telling tester to change region of interest. 
- [x] Test 7 evaluate function kept not succeeding for us and I think this section of the instruction should be a little clearer what is expected. Generally the instructions are descriptive but need to instruct the testers exactly what to do.
- [x] Test 8  should describe the relevant logs to select.
- [x] https://github.com/mantidproject/mantid/issues/34949#issuecomment-1401640186

### PR DESCRIPTION
**Pull request by DanielMurphy22**
**Description of work.**

**To test:**

Read through instructions and check they make sense. I don't think a full manual test of the instructions I've changed is necessary as continuous improvement of our testing instructions each release is the best option.

@RichardWaiteSTFC is my update to Test 8, specifically steps 7-11, not going to work? BBasically, do I need the user to close and reopen the EngDiff gui to enable the sample log settings? 

Fixes #35052 

*This does not require release notes* because **documentation update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst [hunk 1]
```diff
@@ -177,30 +182,60 @@ This tests the operation of the fit browser.
...
+4. In the fit property browser, all the plotted spectra should be available in the Settings > Workspace combo box.
+   In the central ``Run Selection`` table, remove one spectrum from the plot by unticking the ``Plot`` checkbox for one row.
+   The Settings > Workspace combo box should now update and not include the removed spectrum.
 
-4. Perform a fit by clicking Fit>Fit in the fit browser. On completion of the fit, a group workspace with suffix `_fits` should have appeared in the ADS. In this group of workspaces there should be a matrix workspace for each parameter fitted (named by convention FunctionName_ParameterName), to view this right-click on the ADS and `Show Data`. For any runs not fit there should be a NaN value in the Y and E fields. In addition there is a workspace that has converted any peak centres from TOF in d-spacing (suffix `_d`). There should be an additional table called `model` that summarises the chisq value and the function string including the best-fit parameters.
+5. Right-click on the plot image and select Add Peak and add a peak to the plot. Change the peak type to BackToBackExponential and add another peak. Also add a Linear background.
```
Comment (RichardWaiteSTFC) [inline dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst:194]: I thought that the default peak type in the UI was automatically set to `BackToBackExponential` (regardless of what the setting is in the the config)

**Review by RichardWaiteSTFC — CHANGES_REQUESTED** [review]
Looks good, the instructions are much clearer now and it looks like a thorough test of the GSAS tab. Just want some clarification on the default peak function in the UI

File: dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst (hunk 1 above)
Comment (DanielMurphy22) [reply dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst]: My default is for fitting on EngDiff is Lorentzian but I've maybe changed my settings at some stage. I've changed the comment slightly. I would like to ensure that the tester uses a BacktoBackExponential, but hopefully that is already added as their first peak.

**Review by RichardWaiteSTFC — APPROVED** [review]
Looks good, thanks updating this!

**Pull request merged**